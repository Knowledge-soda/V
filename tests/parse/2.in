+(2, *(v(4)), 3)
