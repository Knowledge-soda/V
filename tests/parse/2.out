-1::
  BLN(ID):
    BLN(SUM):
      NUM(2), COM, BLN(MUL):
        BLN(VAL):
          NUM(4), BLN(NOP),
        BLN(NOP),
      COM, NUM(3), BLN(NOP),
    BLN(NOP),

